tests/tt_eager/python_api_testing/sweep_tests/test_configs/ci_sweep_tests/broken_grayskull/pytorch_eltwise_power_fp_test.yaml

```diff
@@ -26,7 +26,7 @@ test-list:
         buffer-type: ["DRAM", "L1", "SYSTEM_MEMORY"]
         out-buffer-type: ["DRAM"]
       output-file: eltwise_power_fp_sweep.csv
```

Comment:
Change to `eltwise_pow_sweep.csv` to match op name changes elsewhere? Also, update the file names from `*_eltwise_power_fp_*` to `*_eltwise_pow__*`

Reply:
@TT-BrianLiu I have updated the sweep tests output file name. 
In order to differentiate between int exponent and float exponent tests i haven't renamed the .yaml files is that fine?

Reply:
is it difficult to rename the yaml as well?